screenpy_selenium/resolutions/is_present.py

```diff
@@ -0,0 +1,19 @@
+"""
+Matches a present WebElement.
+"""
+
+from screenpy.resolutions.base_resolution import BaseResolution
+
+from .custom_matchers import is_present_element
+
+
+class IsPresent(BaseResolution):
+    """Match on a clickable element.
```

Comment:
This docstring should say that it matches a "present" element.

Reply:
whoopsie! fixed